kpops/cli/main.py

```diff
@@ -235,10 +236,10 @@ def generate(
     api_version: Optional[str] = typer.Option(
         None, help="Kubernetes API version used for Capabilities.APIVersions"
     ),
-    ca_file: Optional[str] = typer.Option(
+    ca_file: Optional[Path] = typer.Option(
         None, help="Verify certificates of HTTPS-enabled servers using this CA bundle"
     ),
-    cert_file: Optional[str] = typer.Option(
+    cert_file: Optional[Path] = typer.Option(
         None, help="Identify HTTPS client using this SSL certificate file"
```

Comment:
Nice catch. Should we also update the helm description? Something like
"Path to..."

Reply:
yeah, good idea. So far it's just the same description that Helm help gives I believe